### DISCUSSION TITLE
Problemas técnicos

### DISCUSSION THREAD

**Discussion by jainerms** [rramosp/ai4eng.v1#77] [00 - General]
Cordial saludo profe, como se encuentra?

Profe, el siguiente correo es para pedir orientación en un caso que me esta ahorrando los colab de los laboratorios, cuando ingreso mi cuenta y contraseña, me sale el siguiente error:
image.png
![image](https://user-images.githubusercontent.com/95552158/144728740-a75a85b3-610a-4dcc-a78b-bd358e80fbf5.png)

Y en el RLXMOOC me sale cargado el curso
image.png
![image](https://user-images.githubusercontent.com/95552158/144728746-714b34ab-bcd6-4690-beab-77e2654c3183.png)

Sera que utilizo otro correo para acceder a resolver los laboratorios o que se podría hacer en este caso?

Muchas gracias por leerme, quedo atento a cualquier información.

**Comment by Felipe-RA** [comment]
Saludos Jainerms

Este problema ocurre cuando un estudiante está matriculado en dos versiones distintas del mismo curso (en tu caso la de 2021 1 y la actual)

Te avisamos en cuanto esté solucionado

**Reply by Felipe-RA** [reply]
Vale!

Ya deberías tenerlo solucionado

Avísanos si tienes algún otro problema
